packages/autocomplete-plugin-algolia-insights/src/__tests__/createAlgoliaInsightsPlugin.test.ts

```diff
@@ -833,7 +890,7 @@ describe('createAlgoliaInsightsPlugin', () => {
           eventName: 'Product Selected from Autocomplete',
           index: 'index1',
           objectIDs: ['1'],
-          positions: [0],
+          positions: [1],
```

Comment:
question: was this assertion wrong before the change?

Reply:
yes, the tests did not wait correctly for the debounced "previous items" to be set. positions should be 1 in all those cases

Reply:
Well spotted!